client/src/components/Form/Elements/FormText.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+    <div>
+        <b-alert v-if="errorMessage" class="mt-2" :show="dismissCountDown" variant="info" @dismissed="resetAlert">
+            {{ errorMessage }}
+        </b-alert>
+        <b-row align-v="center">
+            <b-col>
+                <b-form-textarea
+                    v-if="inputArea"
+                    :id="id"
+                    v-model="currentValue"
+                    class="text-input"
```

Comment:
The `text-input` class does not seem to be defined anywhere.

Reply:
You changed this to `ui-input` [here](https://github.com/galaxyproject/galaxy/pull/13375/commits/1c9bed3419d156ed3e4be9f979480d4e1456dce5) so I'm guess that's what it should be?

Reply:
Yes, I restored it to how it was in current dev: https://github.com/galaxyproject/galaxy/blob/dev/client/src/utils/navigation/navigation.yml#L490, and noticed that it's not used elsewhere.